src/components/pages/Testbook/Testbook.tsx

```diff
@@ -0,0 +1,92 @@
+import {
+  Button,
+  Container,
+  Divider,
+  Group,
+  Space,
+  TextInput,
+  Title,
+} from "@mantine/core";
+import React, { useEffect, useState } from "react";
+import { useTestbookContext } from "../../../context/useTestbookContext";
+import { LOADING_STATUS, OperationEnum } from "../../../reducer/testbook/types";
+
+const UseCases: React.FC = () => {
+  const {
+    state: {
+      testbook: { item: testbookItem, status: testbookStatus },
+    },
+    setTestbook,
+    getTestbook,
+  } = useTestbookContext();
+
+  const [name, setName] = useState("");
+  const [description, setDescription] = useState("");
+  const [version, setVersion] = useState("");
+
+  useEffect(() => {
+    getTestbook();
```

Comment:
I would prefer testbook to be initialized in context component

Reply:
It sounds better, I'm gonna refactor this.